my-app/src/app/main-layout/content-projection/courses-page/courses-page.component.scss

```diff
@@ -1,9 +1,9 @@
 .coursesPage {
-  background-color: grey;
-
-  &__item {
```

Comment:
it is better to right coursesPage__item instead of &__item
cause using direct search you would never find coursesPage__item in the current case.
also, we usually use kebab case or bem for class naming